### ISSUE TITLE
PowerPC support

### ISSUE THREAD

**Issue by lu-zero** [nokia/nperf#7]
Currently `nwind` does not support PowerPC, from what I could see to support a new architecture `src/arch/${arch}.rs` and `src/arch/${arch}_get_regs.s` are the main components.

[here](http://openpowerfoundation.org/wp-content/uploads/resources/leabi/content/ch_preface.html) the ABI specification.

**Comment by koute** [comment]
That's correct. In general as long as a given architecture uses DWARF instead of something custom (\*cough\* ARM \*cough\*) it should be pretty straightforward to add support for a new architecture.
